integration_test/third_party_apps_data/applications/mongodb3.6/metadata.yaml

```diff
@@ -27,7 +27,7 @@ minimum_supported_agent_version:
   logging: 2.10.0
 supported_operating_systems: linux
 platforms_to_skip:
-  # MongoDB3.6 hit end of life in 2021 and is not supported on Rocky Linux 9, Ubuntu 20.04, Ubuntu 22.04, Ubuntu 23.04, Debian 10, Debian 11, or Debian 12
+  # MongoDB3.6 hit end of life in 2021 and is not supported on Rocky Linux 9, Ubuntu 20.04, Ubuntu 22.04, Ubuntu 23.04, Debian 10, Debian 11, Debian 12, or SLES 15 AARCH64
```

Comment:
we could amend the comment to something like:

# MongoDB3.6 hit end of life in 2021 and is not supported on various distros.

then we don't have to update it each time.

Same for the previous file (mongodb/metadata.yaml) and probably other subsequent files in this PR.

Reply:
@martijnvans I'm ok with this. Do you want me to wait for a 2nd Google opinion before I try to make these changes?

Reply:
no need to wait for a 2nd google opinion.